### PR TITLE
fixes #33 - validation errors on first page relating to date

### LINKED ISSUE #33: First page of example shows incorrect validation errors
The date `fieldset` includes the following fields, this causes the invalid red left border to show next to irrelevant fields. All validations are on the parent date field, but should be added to the day/month/year children. Also the date ignores the `required` validation.

### PR DESCRIPTION
**Pull request by joefitter**
#33 Removed irrelevant fields from date fieldset, added validations to date children, removed overridden validateField method in controller and added note

### CODE REVIEW

File: apps/my_awesome_form/controllers/first-page.js [hunk 1]
```diff
@@ -11,8 +11,12 @@ var FirstPageController = function FirstPageController() {
 
 util.inherits(FirstPageController, DateController);
 
-FirstPageController.prototype.validateField = function validateField(keyToValidate, req) {
-  return DateController.prototype.validateField.call(this, keyToValidate, req, false);
-};
+/*
+ * The validateField method can be overuidden in the following way to
+ * always allow date to be black, regardless of settings in fields directory
```
Comment (easternbloc) [inline apps/my_awesome_form/controllers/first-page.js]: what does date to be black mean?

File: apps/my_awesome_form/controllers/first-page.js [hunk 2]
```diff
@@ -11,8 +11,12 @@ var FirstPageController = function FirstPageController() {
 
 util.inherits(FirstPageController, DateController);
 
-FirstPageController.prototype.validateField = function validateField(keyToValidate, req) {
-  return DateController.prototype.validateField.call(this, keyToValidate, req, false);
-};
+/*
+ * The validateField method can be overuidden in the following way to
```
Comment (easternbloc) [inline apps/my_awesome_form/controllers/first-page.js]: typo

File: apps/my_awesome_form/controllers/first-page.js (hunk 1 above)
Comment (joefitter) [inline apps/my_awesome_form/controllers/first-page.js]: I cant type obviously :(

Comment (joefitter) [inline apps/my_awesome_form/controllers/first-page.js]: was supposed to be blank, but I guess omitted would have been a better word :)

**Comment by easternbloc** [comment]
Could you rebase the typo commit into one please

**Comment by JoeChapman** [comment]
@joefitter a rebase should fix this build

**Comment by easternbloc** [comment]
@joefitter please reword your commit message to remove `merge in master`

**Pull request merged**